README.md

```diff
@@ -119,6 +119,9 @@ Additionally, `cowboy_swagger` can be configured/customized from a `*.config` fi
  %% cowboy_swagger config
  {cowboy_swagger,
   [
+   %% `path`: Path where you can access Swagger-UI. Default: `/api-docs`.
```

Comment:
```suggestion
   %% `path`: Path where you can access Swagger-UI. Default: `/api-docs`.
   %%         By default will be set value `/api-docs`.
```

Reply:
@vkatsuba I'm not sure what you wanted to add with your comment. The fact that the default value is `/api-docs` was already established in the original text.

Reply:
ops... missed it...